server/routers/tokensRouter.ts

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import Joi from 'joi';
+import TokensModel from 'models/Tokens';
```

Comment:
```suggestion
import TokensModel from 'models/tokens';
```

This caused a build error for me locally